web3.js/src/transaction.js

```diff
@@ -432,38 +444,41 @@ export class Transaction {
    * Verify signatures of a complete, signed Transaction
    */
   verifySignatures(): boolean {
-    return this._verifySignatures(this.serializeMessage());
+    return this._verifySignatures(this.serializeMessage(), true);
   }
 
   /**
    * @private
    */
-  _verifySignatures(signData: Buffer): boolean {
-    let verified = true;
+  _verifySignatures(signData: Buffer, requireAllSignatures: boolean): boolean {
     for (const {signature, publicKey} of this.signatures) {
+      if (signature === null) {
+        return !requireAllSignatures;
```

Comment:
Should be a `continue` if `requireAllSignatures === false`

Reply:
doh, fixed